server/sonar-web/src/main/js/app/styles/components/modals.css

```diff
@@ -30,7 +30,6 @@
   border-radius: 3px;
   top: 15%;
   max-height: calc(100vh - 15%);
-  overflow-y: auto;
```

Comment:
@anup-rabit This value `overflow-y: auto` is required to display vertical scrollbar when there is no enough space on the screen. So this fix is bad for us - because we need scroolbar to appear on all popups except "Add conditions" popup.